apps/src/templates/certificates/petition/ControlledFieldGroup.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {FormControl, FormGroup, HelpBlock} from 'react-bootstrap';
```

Comment:
@tim-dot-org I am curious if you have a sense of when we should be using react boostrap vs not? I think it's fine for this, but up to this point AFAIK we were only really using it in PLC (now EP Tools) land. @megcrenshaw , let's check in before adding react-bootstrap in any new places (beyond this one) outside of existing PLC stuff.

Reply:
Yeah, I would lean towards not using it in more places. We're currently on an older version of react-bootstrap that is going to have less support than the latest version. our web component library should replace most of the components we use in react-bootstrap or else layout type things are implementable using flex/grid.

Reply:
but yea, agree with dave re: its fine for now, since there's not really an obvious replacement yet

Reply:
Sounds good––it was a bit painful finding the old react bootstrap documentation, so it certainly wasn't ideal